scripts/setup-pg_hba.sh

```diff
@@ -12,24 +12,35 @@ fi
 # Reconfigure pg_hba if environment settings changed
 cat ${ROOT_CONF}/pg_hba.conf.template > ${ROOT_CONF}/pg_hba.conf
 
+if [[ "$FORCE_SSL" =~ [Tt][Rr][Uu][Ee] ]]; then
+  PG_CONF_HOST='hostssl'
+  CERT_AUTH='cert'
```

Comment:
Doesn't cert auth uses full verified certificate chain?
How are we going to test this?

Reply:
You can connect the service and call `psql "dbname=gis port=5432 user=docker host=localhost sslmode=verify-full sslcert=/etc/ssl/certs/ssl-cert-snakeoil.pem sslkey=/etc/ssl/private/ssl-cert-snakeoil.key sslrootcert=<path>/root.crt"

Reply:
full verified is client side option not server side